src/util.cpp

```diff
@@ -427,6 +427,19 @@ std::vector<std::string> ArgsManager::GetArgs(const std::string& strArg)
     return {};
 }
 
+std::vector<std::string> ArgsManager::GetUniqueArgs(const std::string& strArg)
+{
+    LOCK(cs_args);
+    std::vector<std::string> args;
```

Comment:
I think just make sure there's a comment at the declaration saying that ordering is stable.